controller/sync.go

```diff
@@ -56,6 +56,18 @@ func (m *appStateManager) getGVKParser(server string) (*managedfields.GvkParser,
 	}
 	return cluster.GetGVKParser(), nil
 }
+func (m *appStateManager) getResourceOperations(server string) (kube.ResourceOperations, func(), error) {
```

Comment:
Want to add a comment to this? "Resource operations" as a phrase doesn't really mean anything to me, a comment might help clarify what this function does.

otoh `kube.ResourcesOperations` might just be a term of art that the reader (me in this case) needs to learn.

Reply:
`ResourceOperations` is an interface defined in gitops-engine that provides methods to manage k8s resources:
https://github.com/argoproj/gitops-engine/blob/c0c2dd1f6f487c46a25d3d3caaad7fec06667728/pkg/utils/kube/resource_ops.go#L40

I will add the godoc to the function.

Reply:
added.